stdlib/runtime/malloc.gr

```diff
@@ -17,12 +17,12 @@ import WasmI32, {
   shl as (<<),
   shrU as (>>),
   eq as (==),
-  ne as (!=)
+  ne as (!=),
 } from "runtime/unsafe/wasmi32"
 import Exception from "runtime/exception"
 
-primitive memorySize: () -> WasmI32 = "@wasm.memory_size"
-primitive memoryGrow: (WasmI32) -> WasmI32 = "@wasm.memory_grow"
+primitive memorySize : () -> WasmI32 = "@wasm.memory_size"
+primitive memoryGrow : WasmI32 -> WasmI32 = "@wasm.memory_grow"
```

Comment:
nitpick but the space of the annotation here isn't consistent with everything else

Reply:
Good catch! We actually have always written primitives with the space, so the others didn't show in the diff. I just pushed a re-grainfmt.